server/gitrest/Dockerfile

```diff
@@ -15,6 +15,7 @@ WORKDIR /home/node/server
 COPY package*.json ./
 COPY pnpm*.yaml ./
 COPY scripts/*.* ./scripts/
+COPY .npmrc ./
```

Comment:
I don't know docker well, and couldn't reproduce the error CI hit locally, so this "fix" should get extra careful review.